qubes/vm/appvm.py

```diff
@@ -120,7 +120,9 @@ def on_domain_loaded(self, event):
     def on_property_pre_reset_template(self, event, name, oldvalue=None):
         '''Forbid deleting template of running VM
         '''  # pylint: disable=unused-argument
-        raise qubes.exc.QubesValueError('Cannot unset template')
+        default_template = getattr(self.app, 'default_template')
```

Comment:
You have to have a default value here, otherwise this will raise an exception if attribute is not found (and tests fail on this).

Reply:
It is possible to have the `default_template` unset? I did not know that.